docs/source/Quickstart/04_non-virt.md

```diff
@@ -57,11 +58,26 @@ pre_infra:
 about 30G of RAM, 10 Cores and 120G of disk space. It's more than the minimal
 requirements.
 
-### 4. Run the playbook
+### 4. Place a pull secret in place
+
+Get the pull secret from `https://cloud.redhat.com/openshift/create/local`
+and place it to `$HOME/pull-secret`.
+
+### 5. Verifiy ansible.cfg
+
+install_yamls/devsetup/ci-framework/ansible.cfg
+The role_path needs to include cifmw_basedir .
+example:
+`roles_path = ~/ci-framework-data/artifacts/roles:./ci_framework/roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles`
+
+You might want to remove the ~/ci-framework-data  directory before rerun.
```

Comment:
we don't want to remove it, but ensure it's present. The generated install_yamls_makes role is there.

Reply:
Moved to the cleanup section.
This was about full reset.